### ISSUE TITLE
Dynamic Variables $NOW Adjustments

### ISSUE THREAD

**Issue by NilsBaumgartner1994** [directus/directus#24207]
### Describe the Request

https://docs.directus.io/reference/filter-rules.html#dynamic-variables

```
$NOW(<adjustment>) - The current timestamp plus/minus a given distance, for example $NOW(-1 year), $NOW(+2 hours)
```

is it possible to set NOW to a specific hour? like adjust minus the current hour + 8 ==> set to 08:00 ?

### Maintenance Strategy

Only when $NOW changes

**Comment by rijkvanzanten** [comment]
No, you can't override a part of the generated timestamp at this point
